src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleFreeLook.kt

```diff
@@ -39,6 +38,14 @@ object ModuleFreeLook : Module(
     override fun enable() {
         cameraYaw = player.yaw
         cameraPitch = player.pitch
+        mc.options.perspective = THIRD_PERSON_BACK
```

Comment:
Why are we changing perspective?

Reply:
So that when FreeLook is disabled by changing perspective, it changes to `THIRD_PERSON_FRONT` instead of some other perspective.

Reply:
I did it so the perspective value in GameOptions is consistent with the actual perspective but then normally disabling the module doesn't reset to the previous perspective. So you would need to store it somehow or leave it how it is and don't allow changing perspective while in FreeLook.